libs/designsystem/src/lib/components/segmented-control/segmented-control.component.scss

```diff
@@ -21,32 +21,62 @@
     }
   }
 
+  &.default-mode {
+    ion-segment {
+      --background: #{utils.get-color('white')};
+
+      display: inline-flex; // The segmented control itself is block-level, but we don't want the background to stretch
+      width: auto; // The segmented control itself is block-level, but we don't want the background to stretch
```

Comment:
Just out of curiosity: Could the same be achieved by using `width: fit-content`?

Reply:
> Just out of curiosity: Could the same be achieved by using `width: fit-content`?

https://github.com/kirbydesign/designsystem/pull/2417#discussion_r949005332